### PR TITLE
`ultralytics 8.3.55` New Medical-pills dataset

### PR DESCRIPTION
**Pull request by RizwanMunawar**
@glenn-jocher Hi, this PR adds a new dataset `medical-pills` in [Ultralytics docs](https://docs.ultralytics.com/). Thank you! 💊

### Train

```python
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n.pt")  # load a pretrained model (recommended for training)

# Train the model
results = model.train(data="medical-pills.yaml", epochs=100, imgsz=640)
```

### Predict

```python
from ultralytics import YOLO

# Load a model
model = YOLO("path/to/best.pt")  # load a fine-tuned model

# Inference using the model
results = model.predict("https://ultralytics.com/assets/medical-pills-sample.jpg")
```

val_batch0_pred

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary  
A new **Medical Pills Detection Dataset** has been added to Ultralytics documentation and repository, tailored for AI applications in the pharmaceutical industry.  

### 📊 Key Changes  
- **New Dataset Added**:  
  - The "Medical Pills" dataset includes 92 training images and 23 validation images annotated for identifying pills.  
  - YAML configuration file (`medical-pills.yaml`) created for seamless integration and training.  
- **Documentation Updates**:  
  - Detailed docs and usage examples (Python and CLI) for training and inference with the medical-pills dataset.  
  - Added the dataset to index files and navigation menus in the documentation.  
- **Version Update**: Bumped version from `8.3.54` to `8.3.55`.  

### 🎯 Purpose & Impact  
- **Purpose**: To support AI research and automation in pharmaceutical applications such as pill sorting, quality assurance, and inventory management.  
- **Impact**:  
  - Enables easy adoption of AI for healthcare-specific tasks through a ready-to-use dataset.  
  - Facilitates training and testing of advanced YOLO models in a niche industry.  
  - Contributes to better accuracy and efficiency in medical workflows via computer vision.  
🌟 Perfect for researchers and developers working in AI, healthcare, or pharmaceutical automation.

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @RizwanMunawar, thank you for submitting an `ultralytics/ultralytics` 🚀 PR! To ensure a seamless integration of your work, please review the following checklist:

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and reference any relevant issues if applicable. Ensure your commit messages are clear, concise, and follow the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `main` branch of the repository. If it’s behind, please update it to avoid potential merge conflicts.
- ✅ **Ensure CI Checks Pass**: Verify that all Continuous Integration (CI) checks are passing for your PR. If there are any failures, investigate and address them before requesting a review.
- ✅ **Update Documentation**: Ensure the documentation reflects any changes or additions you’ve made, so others can easily understand and use your contribution.
- ✅ **Add Tests**: Where applicable, include or update test cases to validate the new functionality or bug fixes. Verify all tests pass successfully.
- ✅ **Sign the CLA**: Make sure you have signed the Contributor License Agreement (CLA) if this is your first contribution. You can do this by stating, "I have read the CLA Document and I sign the CLA" in a comment.
- ✅ **Minimize Changes**: Keep changes concise and limit them to the intended scope of your PR. Reducing unnecessary modifications ensures a smoother review process. _"It's not the daily increase but daily decrease. Hack away at the unessential."_ — Bruce Lee

This is an automated response to help you streamline your submission. A member of the Ultralytics team will review your PR soon to provide further feedback or approval. If you have any questions, don't hesitate to ask. Thank you for contributing to Ultralytics! 🚀✨

**Comment by glenn-jocher** [comment]
@RizwanMunawar could we update the dataset name to maybe 'medical-pills' or just 'pills' to keep it simpler? Thanks!

**Comment by RizwanMunawar** [comment]
@glenn-jocher nice idea! I have updated the medicinal-pill to medical-pills, looks good. Thank you!

**Comment by UltralyticsAssistant** [comment]
🎉 The PR introducing the **Medical Pills Detection Dataset** is officially merged! Huge thanks to @RizwanMunawar for initiating this impactful addition and @glenn-jocher for your expert contributions. Your work is a step forward in merging AI innovation with healthcare transformation. 🚀🩺  

This dataset empowers developers and researchers with tools to address real-world challenges in pharmaceuticals, from automation to quality assurance—work that truly matters.  

As Leonardo da Vinci once said, *"Knowledge of the past and present is the key to the future."* Your collaboration here paves the way for advancements that will shape a better, healthier future. Thank you for making this possible! 🌟
